etsin_finder/frontend/js/stores/view/qvain/index.js

```diff
@@ -253,10 +254,21 @@ class Qvain extends Resources {
     if (this.Env.metaxApiV2) {
       await this.Files.openDataset(dataset)
     }
+  }
 
+  @action editDataset = async dataset => {
+    this.original = { ...dataset }
+    this.loadBasicFields(dataset)
+    this.loadStatusAndFileFields(dataset)
     this.setChanged(false)
   }
 
+  @action resetWithTemplate = async dataset => {
```

Comment:
Ok.. Actually resetWithTemplate doesn't use identifier tested outside of this function call. So some deep in the this.loadBasicFields call is probably identifier usage. It gets kind of hard to discover what is the point of the test in table.jsx.